layouts/partials/home/hero.html

```diff
@@ -20,10 +20,13 @@
           <p class="is-size-3-desktop is-size-4-tablet is-size-5-mobile has-text-weight-light">
             {{ $subtagline }}
           </p>
+        
+          <br/><br/>
+          <p class="is-size-4-desktop is-size-5-tablet is-size-6-mobile has-text-weight-bold">
```

Comment:
I suggest that we add a description of what OpenTelemetry provides (libraries, agents, etc.) before previewing the latest blog post. Here's a modified version of our FAQ text that I propose we use:

OpenTelemetry provides a single set of APIs, libraries, agents, and collector services that capture distributed traces and metrics from your application and send them to backends like Prometheus and Jaeger for processing.

Reply:
Done -- also pushed the blog content down a row. We can add more content to this area later, but the hero was getting full.